packages/app/config/webpack.dev.js

```diff
@@ -18,7 +18,7 @@ module.exports = merge(
   },
   commonConfig,
   {
-    devtool: 'eval',
+    devtool: 'cheap-module-source-map',
```

Comment:
Is React the only known affected library?
https://reactjs.org/docs/cross-origin-errors.html#source-maps

Reply:
@lbogdan 
I revert the commit changing the webpack config:
https://github.com/CompuIves/codesandbox-client/pull/1711/commits/61f828e282cc07102063a0c5d97339f6d0a75104